module/batch-action/src/Domain/Repository/BatchActionRepositoryInterface.php

```diff
@@ -22,5 +23,5 @@ public function addEntry(BatchActionId $id, AggregateId $resourceId): void;
 
     public function markEntryAsSuccess(BatchActionId $id, AggregateId $resourceId): void;
 
-    public function markEntryAsUnsuccess(BatchActionId $id, AggregateId $resourceId, string $message): void;
+    public function markEntryAsUnsuccess(BatchActionId $id, AggregateId $resourceId, Relationship $relationship): void;
```

Comment:
Wondering whether this is the right approach. Potentially unsuccessful operation might be a result of something different?